docs/deployment/feature_gates.md

```diff
@@ -108,6 +106,9 @@ The following tables are a summary of the feature gates that you can set on diff
 | ShootSARotation                              | `true`  | `Beta`       | `1.51` | `1.56` |
 | ShootSARotation                              | `true`  | `GA`         | `1.57` | `1.59` |
 | ShootSARotation                              |         | `Removed`    | `1.60` |        |
+| ReversedVPN                                  | `false` | `Alpha`      | `1.22` | `1.41` |
+| ReversedVPN                                  | `true`  | `Beta`       | `1.42` | `1.61` |
```

Comment:
```suggestion
| ReversedVPN                                  | `true`  | `Beta`       | `1.42` | `1.62` |
```

Reply:
If you take a look at the other lines of this table you will see that the last column is meant to be inclusive. A feature cannot be in `Beta` until (inclusive) 1.63 and in `GA` starting with 1.63. The first version should be less similar to the other feature gates.